src/chapters/aspdotnet-model-classes/model-class.rst

```diff
@@ -0,0 +1,214 @@
+This new ``Event`` class will include a property for ``Name``, a constructor, and a ``ToString()`` override.
+
+In ``Models/Event.cs``:
+
+.. sourcecode:: c#
+   :linenos: 
+
+   namespace CodingEventsDemo.Models
+   {
+      public class Event
+      {
+         public string Name { get; set; }
+
+         public Event(string name)
+         {
+            Name = name;
+         }
+
+         public override string ToString()
+         {
+            return Name;
+         }
+      }
+   }
+
+
+We now need to update the ``POST`` handler that creates new events. 
+
+First, replace the ``Events`` dictionary with a list of ``Event`` objects.
+
+.. sourcecode:: c#
+   :lineno-start: 15
+
+   static private List<Event> Events = new List<Event>();
+
+Update the ``Add()`` method inside of 
+``NewEvent()`` to add a new ``Event`` instance to the list:
+
+.. sourcecode:: c#
+   :lineno-start: 30
+
+   [HttpPost]
+   [Route("Events/Add")]
+   public IActionResult NewEvent(string name)
+   {
+      Events.Add(new Event(name));
+
+      return Redirect("/Events");
+   }
+
+
+Back in ``Events/Index.cshtml``, update the HTML to use the ``Event`` object’s fields, rather than strings.
+
+.. sourcecode:: guess
+   :lineno-start: 22
+
+   @foreach (var evt in ViewBag.events)
+   {
+      <tr>
+         <td>@evt.Name</td>
+      </tr>
+   }
+
+.. admonition:: Tip
+
+   Here's a shorthand to create auto-implementing properties. In a class, type the word “prop” followed 
+   by hitting the Tab key twice. This swiftly supplies the property’s scaffolding:
+
+   .. sourcecode:: c#
+
+      public object MyProperty { get; set; }
+
+
+Add a Model Property - Video
+----------------------------
+
+.. TODO: Add adding model property video
+
+.. topics covered: add description property to event model, update index view and post handler 
+.. to display the property and use it to create the object
+
+YOUTUBE VIDEO HERE
+
+.. admonition:: Note
+
+   The starter code for this video is found at the `create-model branch <https://github.com/LaunchCodeEducation/CodingEventsDemo/tree/create-model>`__
+   of ``CodingEventsDemo``. The final code presented in this 
+   video is found on the `add-property branch <https://github.com/LaunchCodeEducation/CodingEventsDemo/tree/add-property>`__.
+
+Add a Model Property - Text
+---------------------------
+
+To round out the ``Event`` class, we'll add a ``Description`` property to showcase what our events are all about.
+
+Updates to ``Models/Event.cs``:
+
+.. sourcecode:: c#
+   :linenos:
+
+   namespace CodingEventsDemo.Models
+   {
+      public class Event
+      {
+         public string Name { get; set; }
+
+         public Event(string name)
+         {
+            Name = name;
+         }
+
+         public override string ToString()
+         {
+            return Name;
+         }
+      }
+   }
+
+Now that our data is object-oriented, it’s quick and easy to add a new property affiliated with an event. 
```

Comment:
I would rephrase this to "Now that we are using the ``Event`` model, it's ..."

Reply:
i like this reference to oop - it reinforces the notion that models are just objects and emphasizes the relevance of the oop chapters that come before this